docs/configuration/memory_usage.md

```diff
@@ -0,0 +1,146 @@
+[](){ #reducing-memory-usage }
```

Comment:
2 things:
- is it necessary to put these at the top of the file? Can we not just link to the file directly?
- if the title isn't the first line, the nav cannot see it. If you use the title front matter, it can